core/sr-primitives/src/sr_arithmetic.rs

```diff
@@ -160,6 +160,8 @@ macro_rules! implement_per_thing {
 		{
 			type Output = N;
 			fn mul(self, b: N) -> Self::Output {
+				use core::ops::Div;
```

Comment:
```suggestion
				use rstd::ops::Div;
```

preferably all at the top.